### DISCUSSION TITLE
Is Inertia dead?

### DISCUSSION THREAD

**Discussion by rdgout** [inertiajs/inertia#1267] [Help] [answered]
There has not been a new release since the beginning of this year whilst plenty has happened. 
Some stuff was merged after months of waiting but it still hasn't been released..

It seems like Inertia has had a pretty heavy push by the Laravel community but after almost 9 months without an update, explanation what's going on or release I have gotten a strong feeling that all current maintainers are not really working on this at all.

Anyone else have thoughts about this?

Is it maybe a good idea if more maintainers try to step up if anyone is willing?

**Comment by reinink** [comment, marked as answer]
Hey! Nope, not dead, just most of the development has been behind the scenes recently. Here's an update I shared earlier in August in our Discord channel:

<img width="951" alt="image" src="https://user-images.githubusercontent.com/882133/187195619-1bb92560-ce58-41ac-a45b-709a3a628a81.png">

https://discord.com/channels/592327939920494592/592327939920494594/1007343380264665159

Hoping to finally release some of these changes within the next couple of weeks 👍

**Reply by vitalijalbu** [reply]
Ok but man, it's 2024... ok maybe we can help you with some little money... IDK but pls, we love this product and don't want to migrate to other... 👁️

**Comment by dillingham** [comment]
Worth asking again I think

No releases, tweets or discussion in discord in 5-6 months kind of seems like its going to be abandoned

**Reply by vitalijalbu** [reply]
Also asking... :D

**Reply by rdgout** [reply]
Taylor wrote an update in July of 2023 that Inertia is in essence stable and we should not be expecting much more new features.
Any updates should be written for the adapter packages like for example if you want the modals etc.

I enjoy using Inertia and I hope it continues to get (minor) updates to fix issues or upcoming issues with package compatibility but we should not hope for new features that @reinink previewed in the past which are not finished.

**Comment by zcuric** [comment]
@reinink first of all, thanks for Inertia, it's a great tool and we are using it extensively. We also started poking around the code base and started fixing some bugs that we are experiencing and we submitted a PR and contribute to discussion on other PR.

We are kindly asking for Inertia maintainers to at least give an update about state of Inertia and its future. And maybe check the PRs that address security concerns.

Thank you! 🙏

**Reply by stefanocurnis** [reply]
Inertia is slowly getting abandoned. Feel like it's getting killed anytime soon. It's crazy that they focus on Livewire instead of this.

**Reply by ramonmalcolm10** [reply]
Very sad to see, Inertia has so much potential. Not sure if they fully understand the magnitude of what they offer with this package, it literally gave us Nuxt, Next and Sveltekit.

**Reply by deleugpn** [reply]
I think it's relevant to consider what is going on with xz backdoor and be mindful about this conversation. Reinink made a great thing here but has now sort of moved on and its in everyone's right to do so.
Perhaps instead of focusing on the sad part of Inertia, maybe we, as a community, should take a look at the work being done at https://hybridly.dev/ and embrace it? The more community members that adopt it, the higher the chances of it succeeding. Its a win-win situation. Inertia has been great so far and perhaps was just a stepping stone for something even greater.

**Reply by ramonmalcolm10** [reply]
I like the work being done at Hybridly, only concern I had the last time I check, it wasn’t frame agnostic, it only supported Vue and at that point in time it didn’t support SSR. I would much rather a solution that offer what we already have with further enhancement and bug fixes.

**Reply by deleugpn** [reply]
Perhaps you'll have to fork Inertia yourself, then?

**Reply by ramonmalcolm10** [reply]
Would love to do that but currently not at the required skillset level to implement the stuff I would like to implement such as prefetching, streaming etc.
If any members of the community at that level and willing to work on it then am all down for it.

**Reply by ramonmalcolm10** [reply]
I will look at it this weekend though, if am able to add any serious value to the existing work then I will see where it goes from there.

**Reply by stefanocurnis** [reply]
Of course the Laravel team has the right to abandon it and it's fine. This situation of unchecked PRs, 4 months for a critical dependencies bump, the lack of love from the team, and the unclear future is what is making the community nervous. 

Please tell us the future of this thing so we can unblock it, fork it, make longer plans on it, or adopt something else.

**Comment by msurdi** [comment]
Just found [inertiajs-revamped](https://github.com/inertiajs-revamped/inertia) ([site](https://inertiajs-revamped.com/)). Looks promising.

**Comment by laltu-das** [comment]
Try [Hybridly](https://hybridly.dev/guide/comparison-with-inertia.html). Hybridly's purpose is to drastically improve the productivity and the developer experience of writing interactive applications using Laravel and Vue.

**Reply by j-vitali** [reply]
noo wayy 💯

**Reply by j-vitali** [reply]
this happens when no one replies.... 🫰

**Reply by harryqt** [reply]
no support for Svelte. 😕
